inference-engine/tests_deprecated/fluid_preproc/common/fluid_tests.cpp

```diff
@@ -678,6 +679,65 @@ TEST_P(ConvertDepthTestGAPI, AccuracyTest)
         EXPECT_LE(cv::norm(out_mat_ocv, out_mat_gapi, cv::NORM_INF), tolerance);
     }
 }
+
+TEST_P(DivCTestGAPI, AccuracyTest)
+{
+    const auto params       = GetParam();
+    const int in_depth      = std::get<0>(params);
+    const int in_channels   = std::get<1>(params);
+    const cv::Size sz       = std::get<2>(params);
+    const cv::Scalar C      = std::get<3>(params);
+    double tolerance        = std::get<4>(params);
+
+    const int in_type = CV_MAKETYPE(in_depth,in_channels);
+
+    initMatrixRandU(in_type, sz, in_type);
+
+    // G-API code
+    DivCComputation cc(to_test(in_mat1), to_test(out_mat_gapi), to_test(C));
+    cc.warmUp();
+
+    // OpenCV code /////////////////////////////////////////////////////////////
+    {
+        out_mat_ocv = in_mat1 / C;
+    }
+    // Comparison //////////////////////////////////////////////////////////////
+    {
+        EXPECT_LE(cv::norm(out_mat_ocv, out_mat_gapi, cv::NORM_INF), tolerance);
```

Comment:
as far i remember `EXPECT_LE` doesn't generate some fatal exception and make tests passed for FAIL also.
maybe ASSERT_LE is expected here?

Reply:
ASSERT_XXX(): If assertion fails then processing of test terminate.
EXPECT_XXX(): nonfatal failure, allowing processing to continue.
Here it is not a significant difference.